tests/cggi/boolean_line_vectorizer.mlir

```diff
@@ -0,0 +1,75 @@
+// RUN: heir-opt --cggi-boolean-line-vectorize %s
+
+#encoding = #lwe.unspecified_bit_field_encoding<cleartext_bitwidth = 1>
+!ct_ty = !lwe.lwe_ciphertext<encoding = #encoding>
+!pt_ty = !lwe.lwe_plaintext<encoding = #encoding>
+
+// CHECK-LABEL: add_one
+// CHECK-COUNT-1: cggi.packed %[[*]], %[[*]] {gates = #cggi.cggi_gate<"and", "xor">} : (tensor<2x!lwe.lwe_ciphertext
```

Comment:
oh no! only one cggi.packed haha.

could you also add a syntax test in tests/cggi/ that tests the syntax of this operation so we have it as documentation? (if you add the verifier would also be good to test that error condition on the gate length)